ui/components/src/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { IconButton, Stack, TextField } from '@mui/material';
+import React, { useState } from 'react';
+import { HexColorPicker } from 'react-colorful';
+import CircleIcon from 'mdi-material-ui/Circle';
+
+interface ColorPickerProps {
+  initialColor?: string;
+  onColorChange?: (color: string) => void;
```

Comment:
What is the purpose of `onColorChange`? Seems like `initialColor` means that component is uncontrolled so I wonder if this can be misleading. Otherwise, it might make sense to use the controlled component paradigm with `value` and `onChange`

Reply:
good call, replaced `initialColor` with `color and `onColorChange` to `onChange`